website_search/models/website_search.py

```diff
@@ -38,14 +38,14 @@ def _do_search_ir_view_page(self):
             """select
                 id from (
                     select
-                           id, page,
+                           id, page, name,
                            lower (array_to_string(
                                 xpath(
                                     '//*//text()', arch_db::xml), '\n'
                                 )) as view_text
                     from ir_ui_view
                     where type='qweb' and page='t') as table_text
-                where table_text.view_text ilike  %s """,
+                where table_text.view_text ilike  %s or name ilike %s""",
```

Comment:
goode idea. You can also expand that by giving a higher priority to pages where the name matches vs a match in the text, because they probably are more relevant

Reply:
And you can go wild by having a secondary ranking criterium based on how often the search string occurs in the text. Check this (and adapt the info) to see how you can make the count part of your SQL search: https://stackoverflow.com/questions/36376410/counting-the-number-of-occurrences-of-a-substring-within-a-string-in-postgresql